C++14/dp/lps_in_O(n).cpp

```diff
@@ -0,0 +1,55 @@
+//Returns Longest Palindrome Subsequence length. 
+//Time Complexity O(n).
+int LPSManchers(string input) {
```

Comment:
In case this is taken from some submission, please mention the link else ignore this comment.

Reply:
No, this isn't taken from anywhere.